src/main/java/core/basesyntax/reader/CsvReaderImpl.java

```
@@ -0,0 +1,21 @@
+package core.basesyntax.reader;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public class CsvReaderImpl implements CsvReader {
+
+    @Override
+    public List<String> read(String pathFile) {
+        try {
+            return Files.readAllLines(Paths.get(pathFile));
+        } catch (IOException e) {
+            System.out.println("Cannot read file:" + pathFile);
+            e.printStackTrace();
+            return Collections.emptyList();
```

This violates checklist item #6: 'Don't ignore exceptions in try-catch blocks... Let's rethrow a RuntimeException with an informative message and exception object.' Instead of printing to the console and returning an empty list, you must rethrow the exception wrapped in a `RuntimeException`.